docs/_includes/sidebar.md

```diff
@@ -18,7 +18,8 @@
 {% endif %}
 
 <ul class="sidebar-nav list-unstyled">
-  <li class="item"><a href="https://www.cyberark.com/products/privileged-account-security-solution/cyberark-conjur/" target="_blank">CyberArk Conjur Enterprise</a></li>
+  <li class="item"><a class=“event-click”
+  id=”side-nav-button-enterprise” href="https://www.cyberark.com/products/privileged-account-security-solution/cyberark-conjur/" target="_blank">CyberArk Conjur Enterprise</a></li>
```

Comment:
The use of curly (special) quotes here is going to cause issues, they should be plain straight quotes:

![image](https://user-images.githubusercontent.com/4164335/31448213-e5ef1f3e-ae71-11e7-9819-d1d14d5dd7f3.png)